shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/main/java/org/apache/shardingsphere/sharding/rule/ShardingRule.java

```diff
@@ -186,7 +186,6 @@ private BindingTableRule createBindingTableRule(final String bindingTableGroup)
     public Collection<String> getAllTables() {
         Collection<String> result = new HashSet<>(getTables());
         result.addAll(getAllActualTables());
-        result.addAll(broadcastTables);
```

Comment:
@tuichenchuxin Why remove broadcastTables from this method?

Reply:
@strongduanmu Because I put this method into `getTables()`, So it's same.